docs/concepts/services-networking/service.md

```diff
@@ -428,30 +428,26 @@ an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is
 cloud provider does not support the feature, the field will be ignored.
 
 #### SSL support on AWS
-For partial SSL support on clusters running on AWS, starting with 1.3 two
-annotations can be added to a `LoadBalancer` service:
+For partial SSL support on clusters running on AWS, the following annotations can be 
```

Comment:
Can we change the text to be more specific (e.g. the backend is using HTTP without TLS), remove `partial SSL support`